src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+  const daysBeforeMinSale = 2;
+  const daysBeforeMaxSale = 6;
+  const middleSale = 20;
+  const maxSale = 50;
+
+  if (days <= daysBeforeMinSale) {
+    return days * rentCost;
+  } else if (days > daysBeforeMaxSale) {
+    return days * rentCost - maxSale;
+  }
```

Comment:
```suggestion
  if (days <= daysBeforeMinSale) {
    return days * rentCost;
  }
  
  if (days > daysBeforeMaxSale) {
    return days * rentCost - maxSale;
  }
```

you don't need `else`, cause `return` finishes the function execution when called